app/services/page-title-list.js

```diff
@@ -7,7 +7,7 @@ function capitalize(key) {
 
 let defaults = {};
 ['separator', 'prepend', 'replace'].forEach(function (key) {
-  if (config.pageTitle && config.pageTitle[key]) {
+  if (config.pageTitle && (config.pageTitle[key] !== null) && (config.pageTitle[key] !== undefined)) {
```

Comment:
NOTE: Adding safety checks is a great idea. I wonder if we can make this logic clearer using a more functional programming style:
```js
let defaults = {};
let {pageTitle = {}} = config;
['separator', 'prepend', 'replace']
  .filter(key => {
    return key in pageTitle && pageTitle[key];
  })
  .forEach(function (key) {
    defaults[`default${capitalize(key)}`] = pageTitle[key];
  });
```

Reply:
I'm not sure if a functional programming style makes it clearer, actually. How about if we go to ember utils, and use isPresent, instead?

Do you think

`if(config.pageTitle && isPresent(config.pageTitle[key]))...`

is clearer?